src/utils.jl

```diff
@@ -111,6 +111,17 @@ function transform(X::AbstractArray, eidx::Dict)
     Y
 end
 
-### TODO move these to GraphSignals ######
-# @functor FeaturedGraph
-# Zygote.@nograd normalized_laplacian, scaled_laplacian
+function check_num_nodes(fg::FeaturedGraph, x::AbstractArray)
```

Comment:
Move this to GraphSignals as well? I mean after merged.

Reply:
yes if this has other uses outside of the checks in the forward passes of the layers here, otherwise no

Reply:
Because there similar interface desgined for this purpose.

https://github.com/yuehhua/GraphSignals.jl/blob/95b7a4d9f2d13c3668ced829c4a06a2de8bea81a/src/featuredgraph.jl#L94

Reply:
ok, moved down under "# move to GraphSignals.jl"